static/js/user_profile.js

```diff
@@ -230,7 +233,7 @@ function handle_remove_stream_subscription(target_user_id, sub, success, failure
             error: failure,
         });
     } else {
-        // Unsubscribed by admin.
+        // Unsubscribed by admin or bot owner if target is bot.
```

Comment:
Let's remove this comment rather than changing it -- it's more confusing than helpful.